tastyworks/tastyworks_api/tw_api.py

```diff
@@ -0,0 +1,647 @@
+
+    Args:
+        token (str): A valid session token
+        symbol (string): A base symbol (or first characters of a symbol) to search for symbols starting with the string
+
+    Returns:
+        The response from the API server including a list of dictionaries containing the symbols that start with
+        the passed symbol string and all the option chains for each of the found symbols
+    """
+
+    if symbol[0] == '/':
+        url = make_api_url('FUTURE_OPTION_CHAINS', symbol=symbol)
+    else:
+        url = make_api_url('OPTION_CHAINS', symbol=symbol)
+
+    resp = await api_request('GET', url, token)
+    return resp
+
+
+"""
+#####################################
+############# ALERTS ################
+#####################################
+"""
+
+
+async def get_quote_alert(token: str) -> Dict:
+    """
+    Get the account quote alerts
+    """
+    url = make_api_url('QUOTE_ALERTS')
+    resp = await api_request('GET', url, token)
+    return resp
+
+
+async def create_quote_alert(token: str, alert_json: dict) -> Dict:
+    """
+    Create a new quote alert
+    """
+    url = make_api_url('QUOTE_ALERTS')
+    resp = await api_request('POST', url, token, json_data=alert_json)
+    return resp
+
+
+async def delete_quote_alert(token: str, alert_id: str) -> Dict:
+    """
+    Delete a quote alert
+    """
+    url = make_api_url('QUOTE_ALERTS', alert_id=alert_id)
+    resp = await api_request('DELETE', url, token)
+    return resp
+
+"""
+#########################################
+############# WATCHLISTS ################
+#########################################
+"""
+
+
+async def get_watchlists(token: str) -> Dict:
+    """
+    Retrieves the account personal watchlists
+    """
+    url = make_api_url('WATCHLISTS')
+    resp = await api_request('GET', url, token)
+    return resp
+
+
+async def create_watchlist(token: str, watchlist_json: dict) -> Dict:
+    """
+    Creating a watchlist requires watchlist name and the JSON payload including the symbols to include
+    TODO: Will fail if watchlist name already exists, we may want to check or catch the error
+    """
+    url = make_api_url('WATCHLISTS')
+    resp = await api_request('POST', url, token, json_data=watchlist_json)
+    return resp
+
+
+async def update_watchlist(token: str, watchlist_name: str, watchlist_json: dict) -> Dict:
+    """
+    Adding and removing a symbol from a list requires the watchlist name and a new JSON payload.
+    """
+    url = make_api_url('WATCHLISTS', watchlist_name=watchlist_name)
+    resp = await api_request('PUT', url, token, json_data=watchlist_json)
+    return resp
+
+
+async def delete_watchlist(token: str, watchlist_name: str) -> Dict:
+    """
+    Deleting a watchlist using its name and a DELETE method
+    """
+    url = make_api_url('WATCHLISTS', watchlist_name=watchlist_name)
+    resp = await api_request('DELETE', url, token)
+    return resp
+
+
+async def get_watchlists_public(token: str) -> Dict:
```

Comment:
Public/Private is actually covered in `tastyworks.models.watchlists.WatchlistGroup.get_watchlists()`

For Public:
```python
watchlists = tastyworks.models.watchlists.WatchlistGroup.get_watchlists(TastyworksAPISession)
```

For Private
```python
watchlists = tastyworks.models.watchlists.WatchlistGroup.get_watchlists(TastyworksAPISession, public=False)
```